packages/circuit-ui/components/ValidationHint/ValidationHint.tsx

```diff
@@ -43,6 +43,15 @@ const validStyles = ({ theme, showValid }: StyleProps & ValidationHintProps) =>
     color: ${theme.colors.success};
   `;
 
+const warningStyles = ({
+  theme,
+  hasWarning,
+}: StyleProps & ValidationHintProps) =>
+  hasWarning &&
+  css`
+    color: ${theme.colors.n800};
```

Comment:
Hm actually I'm not sure this is a good idea, here are my thoughts:

* this means that the `ValidationHint` could be two similar but distinct shades of gray: `n700` (the default, declared in `baseStyles` on line 36) or `n800` (when the warning is there)
* this is adding more complexity to our theme (because right now we can hardcode these colors but semantically, what do they actually mean?)
* if a field with the optional label has a warning, it will have three grays: the label (`n900`), the optional label (`n700`) and the warning (`n800`). This doesn't seem right

So I think we should either embrace our subtle body color (`n700`, the current prod color) or go for `n900` instead to match the label (and in this case, we might want to make this the `ValidationHint` default instead of adding it only to the warning variant).

Reply:
<img width="894" alt="Screenshot 2022-02-02 at 16 58 49" src="https://user-images.githubusercontent.com/35560568/152190266-103e315a-b1bd-46c9-acac-03232f9f40a7.png">

Reply:
Fixed in [91af508](https://github.com/sumup-oss/circuit-ui/pull/1386/commits/91af508ffd583b59d4b5728a22c4cf1b73b5c808)